tuplex/utils/src/Serializer.cc

```diff
@@ -729,7 +729,7 @@ namespace tuplex {
                 }
             }
 
-#ifdef DEBUG
+#ifdef NDEBUG
```

Comment:
actually this should be #ifndef NDEBUG. I.e. if not defined no debug == not no debug == debug